earth2mip/schema.py

```diff
@@ -123,6 +128,11 @@ class EnsembleRun(pydantic.BaseModel):
     # alternative for specifying output
     output_dir: Optional[str] = None
     output_path: Optional[str] = None
+    model_registry: Optional[str] = None
+    world_size: Optional[str] = None
```

Comment:
As per the comment above, world size is the number of accelerators. Would this not be an integer? Or are the libraries used expecting a str?

Reply:
The libraries used are expecting a string, not sure why it was designed that way.